flowzone.yml

```
@@ -2251,8 +2251,10 @@ jobs:
             exit 1
           fi
 
+          # here the number of required approvals sent to the gh API is checked back from the PUT response from GH api 
+          # Specifies the number of reviewers required to approve pull requests. Use a number between 1 and 6 or 0 to not require reviewers (successful checks => merge) .
           if [[ $(echo "$result" | jq '.required_pull_request_reviews.required_approving_review_count') -ne ${{ inputs.required_approving_review_count }} ]]; then
-            echo ::error title={url}::"branch protection required_pull_request_reviews.required_approving_review_count not set to ${{ inputs.required_approving_review_count }}"
+            echo ::error title={url}::"Setting number of reviewers required to approve pull requests failed on branch protection GH API as response preoperty required_pull_request_reviews.required_approving_review_count is not set to ${{ inputs.required_approving_review_count }}"
```

This is still not very helpful? It's just a long way of saying we failed to apply a setting without providing any suggestions of how to fix it.
```suggestion
            echo ::warning title={url}::"Failed to set required approvers count. Check that branch protection is enabled on your repo and try again."
```
We could print a warning instead of exiting, and provide suggestions for how to resolve the warning.